config/csscomb.json

```diff
@@ -63,9 +63,17 @@
             "clip",
             "zoom",
             "flex-direction",
-            "flex-order",
-            "flex-pack",
-            "flex-align"
+            "flex-wrap",
+            "flex-flow",
```

Comment:
`flex-flow` is shorthand for `flex-direction flex-wrap` and should therefore appear before either of these 2 properties so that the order is less specific to more specific